modules/oc-mirror-command-reference-v2.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+// * updating/updating_a_cluster/updating_disconnected_cluster/mirroring-image-repository.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oc-mirror-command-reference-v2_{context}"]
+= Command reference for V2
+
+The following tables describe the `oc mirror` subcommands and flags for V2:
+
+.oc mirror subcommands
+[cols="1,2",options="header"]
+|===
+|Subcommand
+|Description
+
+|`completion`
+|Generate the autocompletion script for the specified shell.
+
+|`describe`
+|Output the contents of an image set.
+
+|`help`
+|Show help about any subcommand.
+
+|`version`
+|Output the oc-mirror version.
+
+|`delete`
+|Used to delete images in remote registry and local cache.
+
+|===
+
+.oc mirror flags
+[cols="1,2",options="header"]
+|===
+|Flag
+|Description
+
+|`-c`, `--config` `<string>`
+|Specify the path to an image set configuration file.
+
+|`--dest-skip-tls`
+|Disable TLS validation for the target registry.
+
+|`--dest-use-http`
+|Use plain HTTP for the target registry.
+
+|`--dry-run`
+|Print actions without mirroring images. Generates `mapping.txt` files.
+
+|`--from <string>`
+|Specify the path to an image set archive that was generated by an execution of oc-mirror to load into a target registry.
+
+|`-h`, `--help`
+|Show the help.
+
+|`--max-nested-paths <int>`
+|Specify the maximum number of nested paths for destination registries that limit nested paths. The default is `0`.
+
+|`--source-skip-tls`
+|Disable TLS validation for the source registry.
+
+|`--source-use-http`
+|Use plain HTTP for the source registry.
+
+|`-v`, `--verbose` `<int>`
+|Specify the number for the log level verbosity. Valid values are `0` - `9`. The default is `0`.
```

Comment:
replaced by --loglevel

List of all v2 flags:
Flags:
      --authfile string        path of the authentication file. Default is ${XDG_RUNTIME_DIR}/containers/auth.json
  -c, --config string          Path to imageset configuration file
      --dest-tls-verify        require HTTPS and verify certificates when talking to the container registry or daemon
      --dry-run                Print actions without mirroring images
      --from string            local storage directory for disk to mirror workflow
  -h, --help                   help for oc-mirror
      --loglevel string        Log level one of (info, debug, trace, error) (default "info")
      --max-nested-paths int   Number of nested paths, for destination registries that limit nested paths
  -p, --port uint16            HTTP port used by oc-mirror's local storage instance (default 55000)
      --secure-policy          If set (default is false), will enable signature verification (secure policy for signature verification).
      --since string           Include all new content since specified date (format yyyy-MM-dd). When not provided, new content since previous mirroring is mirrored
      --src-tls-verify         require HTTPS and verify certificates when talking to the container registry or daemon
      --strict-archive         If set (default is false), generates archives that are strictly less than archiveSize (set in the imageSetConfig). Mirroring will exit in error if a file being archived exceed archiveSize(GB).
  -v, --version                version for oc-mirror
      --workspace string       oc-mirror workspace where resources and internal artifacts are generated

Reply:
Understood. Thanks for putting it together.